### ISSUE TITLE
ValueError: Invalid header value b'<consumer_key>:V=2 / 1.25; Python / 3.5.1 (default, May 23 2016, 09:21:16) \n[GCC 4.9.2];'

### ISSUE THREAD

**Issue by foxmask** [evernote/evernote-sdk-python3#16]
When I try to create a note, I give the consumer key and so on, but with Python 3.5 we got this error .

```python
    self.note_store = self.client.get_note_store()
  File "lib/evernote/api/client.py", line 70, in get_note_store
    note_store_uri = user_store.getNoteStoreUrl()
  File "lib/evernote/api/client.py", line 131, in delegate_method
    )(**dict(list(zip(arg_names, args))))
  File "lib/evernote/edam/userstore/UserStore.py", line 1155, in getNoteStoreUrl
    self.send_getNoteStoreUrl(authenticationToken)
  File "lib/evernote/edam/userstore/UserStore.py", line 1164, in send_getNoteStoreUrl
    self._oprot.trans.flush()
  File "lib/thrift/transport/THttpClient.py", line 108, in _f
    result = f(*args, **kwargs)
  File "lib/thrift/transport/THttpClient.py", line 139, in flush
    self.__http.putheader(key, val)
  File "/opt/python-3.5/lib/python3.5/http/client.py", line 1060, in putheader
    raise ValueError('Invalid header value %r' % (values[i],))
ValueError: Invalid header value b'<my_consumer_key>:V=2 / 1.25; Python / 3.5.1 (default, May 23 2016, 09:21:16) \n[GCC 4.9.2];'
```
This works in Python 3.4.

But since 3.5 there is a new check:

```python 
if _is_illegal_header_value(values[i]):                         
    raise ValueError('Invalid header value %r' % (values[i],))
```

**Comment by rdhyee** [comment]
@foxmask Did you see the fix used by a number of [forks](https://github.com/evernote/evernote-sdk-python3/network) -- e.g., [fix custom http headers · djudman/evernote-sdk-python3@62362a2](https://github.com/djudman/evernote-sdk-python3/commit/62362a2e02bc3f87f374c3261f25d67ff096029c).

I'm about to try it myself.

**Comment by IamGianluca** [comment]
The fix suggested by @rdhyee works for me. Thanks!

**Comment by foxmask** [comment]
Sorry i missed to test as i was on holiday s and i forgot when i came back . I make a try after lunch today  😏

**Comment by foxmask** [comment]
there is a PR about it https://github.com/evernote/evernote-sdk-python3/pull/13 since 7 months. 

we can ask whether Evernote is interested in his own SDK :(

**Comment by IamGianluca** [comment]
I was actuality wondering the same. Both python SDKs don't seem to receive much love from Evernote. Lots of open PRs dated months ago :(

**Comment by akhaku** [comment]
Thanks, sorry guys, merged in #13
